src/game_inventory.cpp

```diff
@@ -401,6 +401,24 @@ item_location game::inv_map_splice( const item_filter &filter, const std::string
                          title, radius, none_message );
 }
 
+class liquid_inventory_filter_preset : public inventory_filter_preset
+{
+    public:
+        explicit liquid_inventory_filter_preset( const item_location_filter &filter ) :
+            inventory_filter_preset( filter ) {
+            append_cell( []( const item_location & loc ) {
+                if( loc.get_item() ) {
+                    units::volume vol = loc.get_item()->max_containable_volume();
```

Comment:
If a container already contains some liquid this does not tell whether it can hold all of the new liquid. Maybe we should also show the remaining storage of the container.

Also, it would be nice to color-code the strings so it's clear whether a container can contain all of the liquid or only some of it, to avoid confusion when liquid volume and container storage are rounded to the same number.

Reply:
max_containable_volume shows the remaining volume not the base volume.

in regards to color coding ... maybe? i don't see a bit advantage in it personally and am a bit afraid that people might misunderstand and feel forced to only select lets say green colored containers or avoid red/yellow colored ones.

Reply:
It can be useful when the liquid and the container volume both round to the same number but the liquid volume is slightly larger than the container volume. Another way is to always round up liquid volume and round down container volume.

Reply:
the rounding idea has the problem that format_volume truncates ... it might be worth changing that, but i think due to how often and widespread it is used it probably is best to do that in a seperate pr. (should make it easier to isolate in case something goes wrong)